src/ajitSJune20/FibonacciSeries.java

```diff
@@ -0,0 +1,27 @@
+package ajitSJune20;
+
+import java.util.Scanner;
+
+public class FibonacciSeries {
+
+	static void printFibonacci(int maxNumber) {
+		int previousNumber = 1;
+		int nextNumber = 1;
+		System.out.println("\nFirst " + maxNumber + " elements in fibonacci series are :");
+		for (int i = 1; i <= maxNumber; ++i) {
+			System.out.print(previousNumber + " ");
+			int sum = previousNumber + nextNumber;
+			previousNumber = nextNumber;
+			nextNumber = sum;
+		}
+	}
+
+	public static void main(String[] args) {
+		System.out.print("Enter Number upto which Fibonacci series to Print:");
+		Scanner scanner = new Scanner(System.in);
+		int maxNumber = scanner.nextInt();
```

Comment:
onlythis variable name maxNumber is somewhat misleading .. as 10, 5 is out count for upto n numbers... rest is all good

Reply:
Suggested Changes incorporated. 
Changed maxNumber variable name  to n